### PR TITLE
Refactor/es6

### PR DESCRIPTION
**Pull request by Spring3**

### CODE REVIEW

File: src/dom-extractor.js [hunk 1]
```diff
@@ -1,169 +1,169 @@
...
+    if (params.url !== undefined && params.selector !== undefined) {
+      params.selector = params.selector.replace('|sharp|', '#');
+      if (params.extraCss !== undefined) {
+        params.extraCss = params.extraCss.replace('|sharp|', '#');
+      }
+      extractor.fetch(params.url, {
+        selector: params.selector,
+        extraCss: params.extraCss,
+        removeLinks: params.removeLinks || false
+      }, function (response) {
```
Comment (ky23) [inline src/dom-extractor.js]: arrow function here

File: test/dom-utils-spec.js [hunk 2]
```diff
@@ -1,68 +1,71 @@
...
+    assert(domutils.relToAbs('https://github.com/a/?param=toto', 'http://bitbucket.com/dial-once.css') === 'http://bitbucket.com/dial-once.css');
   });
-  it("should return correct absolute URL [absolute url without protocol]", function () {
-    expect(domutils.relToAbs('https://github.com/a/?param=toto', '//dial-once.css')).toBe('//dial-once.css');
+  it('should return correct absolute URL [absolute url without protocol]', function () {
+    assert(domutils.relToAbs('https://github.com/a/?param=toto', '//dial-once.css') === '//dial-once.css');
   });
-});
\ No newline at end of file
+});
```
Comment (ky23) [inline test/dom-utils-spec.js]: why you don't use arrow functions here ? any reason ?

File: test/sample.js [hunk 3]
```diff
@@ -1,47 +1,54 @@
...
-	console.log('F:', f);
-});
\ No newline at end of file
+domextractor.fetch('http://www.camif.fr/espace-service/livraison-gratuite.html', {
+  selector: '.contentpconseil',
+  innerText: false,
+  inlineCss: true
+}, function (f) {
+  console.log('F:', f);
+});
```
Comment (ky23) [inline test/sample.js]: same for this file, arrow functions

File: test/dom-utils-spec.js (hunk 2 above)
Comment (Spring3) [inline test/dom-utils-spec.js]: I think I just forgot to change them because of the eslint inline rules.

**Comment by Spring3** [comment]
bump @ky23 @jkernech @mrister

File: test/dom-utils-spec.js [hunk 4]
```diff
@@ -1,68 +1,69 @@
...
+const assert = require('assert');
+
+describe('dom utils URL validation', () => {
+  it('should handle bad parameter [no param]', () => {
+    assert(domutils.isValidUrl() === false);
   });
-  it("should handle bad parameter [null param]", function () {
-    expect(domutils.isValidUrl(null)).toBe(false);
+  it('should handle bad parameter [null param]', () => {
+    assert(domutils.isValidUrl(null) === false);
```
Comment (jkernech) [inline test/dom-utils-spec.js]: `assert.equal` would provide more detailed infos, applicable for all assertions below

File: src/dom-utils.js [hunk 5]
```diff
@@ -1,25 +1,27 @@
...
-utils.relToAbs = function(a, b) {
-	if(a === undefined || b === undefined || b.indexOf('//') === 0){
-		return b;
-	}
-	return url.format(url.resolve(url.parse(a), url.parse(b)));
-};
\ No newline at end of file
+module.exports = Utils;
+
+// see http://stackoverflow.com/questions/7544550/javascript-regex-to-change-all-relative-urls-to-absolute
```
Comment (Spring3) [inline src/dom-utils.js]: But relToAbs is part of the Utils class. I don't think I can insert module.exports inside a class scope.

Comment (jkernech) [inline src/dom-utils.js]: I meant the comment 😉

**Comment by Spring3** [comment]
@ky23 please review once more when you are available.

**Pull request merged**